INaturalistIOS/INaturalistIOS-Info.plist

```diff
@@ -44,11 +44,11 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>180</string>
+	<string>182</string>
 	<key>Fabric</key>
 	<dict>
 		<key>APIKey</key>
-		<string>7baaee0ca8a376b3d129b908c1e03be6ecbe5786</string>
+		<string>345e342e4072648b2f39f3eb6ba91e02002087e3</string>
```

Comment:
I think your own Crashlytics APIKey should remain in your fork. The iNaturalist tree will use the iNaturalist Crashlytics API key.